modules/statics/test/unit/coins.ts

```diff
@@ -30,7 +30,17 @@ coins.forEach((coin, coinName) => {
 
     if (!coin.isToken) {
       it(`has expected network type`, function () {
-        coin.network.type.should.eql(coin.name === coin.family ? NetworkType.MAINNET : NetworkType.TESTNET);
+        if (coin.family !== CoinFamily.FIAT) {
```

Comment:
I think this should be the other way around, so `if (coin.family === CoinFamily.FIAT) {` with our fiat specific tests in that if, instead of if not else. Better prepares for future edge cases, if they arise

Reply:
thanks! I did some clean-up in the tests. I moved the tests related to fiat tokens to the `fiatTokens.ts` file and use `fiat.ts` file to check fiat coins.